style.css

```diff
@@ -0,0 +1,71 @@
+body {
+  background-image: url(peace.jpg);
+  background-repeat: repeat-y;
+  background-size: 70%;
+  position: fixed;
+  top: 10%;
+  left: 70%;
+}
+.box {
+  width: 120%;
+  border-radius: 2%p;
```

Comment:
This unit is falsy. Also, since you've no borders defined then you probably don't need that.